packaging/vagrant/Vagrantfile

```diff
@@ -235,6 +235,8 @@ EOF2
     passwd -l root
     passwd -l vagrant
 
+    # Reboot the machine so the paswordless login takes affect
+    sudo reboot
```

Comment:
Would `c.vm.provision :reload` work here?

Reply:
I was initially using c.vm.provision :reload, but that can't be called inside of the provision since everything in that context has to be bash. Additionally, since the "ova" provisioner is only called conditionally, it wouldn't run after it either. I spent a long time looking for a way to conditionally perform :reload based upon the name of a provision, but there doesn't seem to be one. Therefore, this seems like a simple solution, and it has the intended effect.